docs/source/quickstart.mdx

```diff
@@ -85,4 +85,96 @@ tokenizer = AutoTokenizer.from_pretrained(model_id)
 pipe_cls = pipeline("text-classification", model=model, tokenizer=tokenizer)
 text = "He's a dreadful magician."
 outputs = pipe_cls(text)
+```
+
+# Distributed training
+
+When distillation on a single CPU is too slow, we can use multiple CPUs. This guide focuses on PyTorch-based DDP enabling distributed CPU distillation efficiently.
```

Comment:
I would remove the mention to distillation (as it makes it seems like this option is specific to the distillation case)